site/global-guides/voice/authentication.mdx

```diff
@@ -0,0 +1,75 @@
+
+Bandwidth accepts no responsibility for the loss of account credentials and any resulting network traffic, fraud, or undesired account access that results from failing to manage account access credentials in a completely secure manner.
+:::
+
+
+The process to retrieve an access token requires you to have your [WebRTC Credentials](./inAppGuide.mdx#generating-credentials-for-an-access-token) (ie. ClientID and ClientSecret), generated from the [Global Portal](https://login.voxbone.com/login). You will then be able to make a call to our token endpoint with these credentials. This will return to you an `access_token`, `access_token_id`, `token_type`, and `expires_in` (this is the number of seconds before the token will expire).
+
+```mermaid
+sequenceDiagram
+    Application->>In-App Calling AuthServer: GET /api/v1/oauth2/token?scope=voice.webrtc <br> Headers: { Authorization: Basic PEJhc2ljIEF1dGggVXNlcm5hbWU+OjxCYXNpYyBBdXRoIFBhc3N3b3JkPg== }
+    In-App Calling AuthServer->>In-App Calling AuthServer: Validate credentials
+    In-App Calling AuthServer->>Application: 200 OK <br> {"access_token": "...", "access_token_id": "...", "token_type": "Bearer", "expires_in": 600}
+    Application->>WebRTC Gateway: GET /someresource <br> Headers: {Authorization: <token_type> <access_token>}
+```
+### Request
+In order to retrieve the token, you will use Basic Authentication `ClientID:ClientSecret`. Below is a sample request:
+
+```sh
+curl --request POST \
+  --url https://id.voxbone.com/api/v1/oauth2/token?scope=voice.webrtc \
+  --header 'Authorization: Basic PEJhc2ljIEF1dGggVXNlcm5hbWU+OjxCYXNpYyBBdXRoIFBhc3N3b3JkPg==' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data grant_type=client_credentials
+```
+Now that we've shown the request let's break down the parts of the API call.
+
+- **url** `https://id.voxbone.com/api/v1/oauth2/token?scope=voice.webrtc` - This is the In-App Calling AuthServers's `token_endpoint`. **IMPORTANT** There is a query parameter of `scope`. This is used to limit the permissions in the token. When you request a token with the scope parameter that token will **ONLY** be able to access resources that require the specifiied scopes. In our example, we are limiting the token's permissions to only the WebRTC Gateway. If you do not provide a scope, then the token will be minted for ALL scopes that are allowed by the credentials when they were created.
+- **Headers**
+  - **Authorization** - This request requires authentication. You MUST provide your `ClientID` and `ClientSecret` in a Basic Authorization header. Basic Authentication uses a Base64 encoded string that contains the username and password.
+  - **Content-Type** - For this request we **MUST** to use `application/x-www-form-urlencoded`. This will URL encode (aka percent-encode) our parameters making them into `key=value` pairs which will be separated by an ampersand `&`.
```

Comment:
```suggestion
  - **Content-Type** - For this request you **MUST** use `application/x-www-form-urlencoded`. This will URL encode (aka percent-encode) the parameters making them into `key=value` pairs which will be separated by an ampersand `&`.
```